automated install/uninstall.sh

```diff
@@ -8,7 +8,16 @@
 # This file is copyright under the latest version of the EUPL.
 # Please see LICENSE file for your rights under this license.
 
-source "/opt/pihole/COL_TABLE"
+# Include required constants
+readonly PI_HOLE_FILES_DIR="/etc/.pihole"
+PH_TEST="true"
+# shellcheck source=SCRIPTDIR/basic-install.sh
+source "${PI_HOLE_FILES_DIR}/automated install/basic-install.sh"
+# setupVars set in basic-install.sh
+# shellcheck disable=SC1090
+source "${setupVars}"
+# shellcheck source=advanced/Scripts/COL_TABLE
```

Comment:
This would need to be `../advanced/Scripts/COL_TABLE`  since we're in the subdirectory `automated install` here